tests/tips_views_tests.py

```diff
@@ -72,27 +83,53 @@ def test_bad_form(self, generate_bad_form):
 
 @pytest.mark.django_db
 class TestTipViews:
-    def test_tips_board_function_200(self, client):
-        response = client.get('/tips/')
-        assert response.status_code == 200
-        assert 'Tips' in str(response.content)
-
-    def test_tips_views_templates(self):
-        try:
-            get_template(TIPS_PAGE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f'Template {TIPS_PAGE_ROUTE} does not exist'
-        try:
-            get_template(ADD_TIP_PAGE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f'Template {ADD_TIP_PAGE_ROUTE} does not exist'
-        try:
-            get_template(EDIT_TIP_PAGE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f'Template {EDIT_TIP_PAGE_ROUTE} does not exist'
+    def test_tips_views_tips_template(self):
+        get_template(TIPS_PAGE_ROUTE)
+
+    def test_tips_views_add_tip_template(self):
+        get_template(ADD_TIP_PAGE_ROUTE)
+
+    def test_tips_views_edit_tip_template(self):
+        get_template(EDIT_TIP_PAGE_ROUTE)
+
+    def test_tips_views_read_more_template(self):
+        get_template(READ_MORE_PAGE_ROUTE)
+
+    def test_tips_views_categories_template(self):
+        get_template(CATEGORIES_PAGE_ROUTE)
 
     def test_delete_tip_function(self, client, generate_tip):
-        generate_tip.save()
-        assert generate_tip in Tip.objects.all()
-        client.get('/tips/delete_tip/' + str(generate_tip.id) + '/')
-        assert generate_tip not in Tip.objects.all()
+        test_tip = generate_tip(author=AUTHOR, date=DATE, text=TEXT, category=CATEGORY)
+        test_tip.save()
+        assert test_tip in Tip.objects.all()
+        if User.get_session_auth_hash == test_tip.user_token:
+            client.get('/tips/delete_tip/' + str(test_tip.id) + '/')
+            assert test_tip not in Tip.objects.all()
+        else:
+            assert test_tip in Tip.objects.all()
+
+    # Creates different tips and return the tip to filter by
+    @staticmethod
+    def create_test_tips(generate_tip):
+        test_tip = generate_tip(author=AUTHOR, date=DATE, text=TEXT, category=CATEGORY)
+        test_tip.save()
+        tip_to_filter_by = generate_tip(author=OTHER_AUTHOR, date=OTHER_DATE, text=OTHER_TEXT, category=OTHER_CATEGORY)
+        tip_to_filter_by.save()
+
+        return tip_to_filter_by
+
+    def test_filtered_tips_by_category(self, client, generate_tip):
+        tip_to_filter_by = self.create_test_tips(generate_tip)
```

Comment:
def test_create_tips

Reply:
I think when you ask me to change the name of the function there is non need to mention to change all function call too.
Thank you though.